docs/source/poet/sgx_primer.rst

```diff
@@ -0,0 +1,205 @@
+* It then replaces the MAC with a signature created with a device specific
+  private key stored in the device hardware.
+* This new data structure is called a “quote”.
+
+The Quoting Enclave uses the Intel® Enhanced Privacy ID (EPID) signature scheme
+to overcome the privacy concerns associated with using a small number of
+asymmetric keys for signing over the life of a platform. “EPID is a group
+signature scheme that allows a platform to sign objects without uniquely
+identifying the platform or linking different signatures.” [2]_ In “anonymous”
+mode, a signature verifier cannot associate a given signature with a given
+identity. In “pseudonymous” mode, a signature verifier can determine whether it
+has verified a platform previously.
+
+The remote attestation procedure is roughly the following:
+
+1. A local application enclave wants to interact with a remote enclave so it
+   connects to the remote service through the local application.
+#. The remote service issues a challenge to the local application.
+#. The local application retrieves the local quoting enclave’s identity and
+   passes it into the the local application enclave along with the challenge.
+#. The local application enclave generates a “manifest” that includes:
+
+   a. A response to the challenge
+   #. An ephemeral public key
+
+#. The local application enclave generates a hash digest of the manifest. An
+   enclave report is generated which includes the hash digest and has the
+   platform’s (local) Quoting Enclave as its target enclave.
+#. The enclave report and manifest are sent to the local application which
+   forwards the report to the Quoting Enclave.
+#. The Quoting Enclave verifies the report, generates an enclave quote using its
+   EPID key, and forwards the quote to the local application.
+#. The local application forwards the enclave quote and manifest to the remote
+   service.
+#. The remote service verifies the quote and manifest
+
+   a. An EPID public key certificate along with revocation information can be
+      used to verify both locally
+   #. Alternatively, an attestation verification service can be used (see IAS
+      below).
+
+Intel® Attestation Service
+--------------------------
+
+As an alternative to keeping track of a set of verified and revoked enclave
+identities, Intel® provides an attestation service (IAS) which can be used to
+verify enclave quotes and challenge manifests. Roughly speaking, IAS provides:
+
+* An up-to-date list of revoked EPID signatures (which correspond to processors)
+* Enclave quote verification
+
+Inter-Platform Attestation and IAS are used by PoET when a new validator is
+signing up with a network to verify:
+
+* The validator is running the correct PoET code in a trusted enclave.
+* The enclave is not running on a processor that has been revoked.
+
+Each instance of Sawtooth running PoET must have a unique Service Provider ID
+which is used to communicate with IAS and identifies the network.
```

Comment:
Each Sawtooth blockchain Network should use a unique Service Provider ID. For example, this could be the identity of the consortium. All nodes in that network use the same Service Provider ID in the PoET enclave.  The Service Provider ID is used to authenticate with the Intel Attestation Service.